default/stringtables/en.txt

```diff
@@ -339,11 +339,13 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. You can specify the amount and repetition of the transfer. PP spent on the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only transfers PP that are available at this planet so only PP produced in the same supply-group are used and no PP from the imperial stockpile.
 
-Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used to transfer PP to the stockpile even if there are things on the queue below it that could have been allocated available PP.'''
+Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue and may trigger a warning.  This project is used for fine-tuning allocation of PP to the stockpile even if there are items on the build queue below it that could have been allocated available PP.
+
+A fully funded transfer will be finished in a single turn so if you want to make sure you always have at least 10PP available in the stockpile, you can add a 10x Stockpile Transfer and set the repetition to 99 times.'''
```

Comment:
This sentence is unclear. I suggests that having a 10x Stockpile Transfer will make at least 10 PP available, which is not clearly different from just always transferring up to 10 PP, and it doesn't clarify that it would only work if there is at least 10 PP available on the queue at that position. If you want to give some examples, clearly mark it as such, and give more details about the whole queue.

Reply:
I tried to make the example more clear. Probably the entry should be moved to another pedia article as there is some overflow even though only added one example. And i would like to add another example:

Another example: One has only one supply-group but wants to make sure that 5PP are always available in the stockpile for future use. Lets say one can draw more than 5PP from the stockpile. One can either disable stockpile use for all build items after adding once 5PP to the stockpile. Or one can add a 5x transfer project to the top of the build queue with a 99 times repetition.
This way always 5PP are added to the next turn's imperial stockpile while other projects use up the current stockpile.